### ISSUE TITLE
[Feature Request]: Allow a section to be hidable to show what's on the left

### ISSUE THREAD

**Issue by pskowronek** [jordanbaird/Ice#310]
### Check Existing Issues

- [X] I have checked existing issues, and this issue is not a duplicate

### Description

A feature request: Allow a section to be hidable i.e. not only everything to left, but only part (a section).

Add possibility to select/mark a section of icons that be could be made hidden temporarily.

Use case - having mac 13 or 14" and opened application with wide menu, the menu items disappear on the left side. Another use-case is macbook with notch, when if you have something like Menu Meters and a couple of other menubar apps they quickly disappear under the notch. If there was an option in Ice to collapse a section, then icons on the left could be now visible and accessible when section collapsed. Very helpful if you have Menu Meters (cpu/mem gauges) and still you have some apps that you would like to be able to use.

**Comment by Aaron-Rumpler** [comment]
Agreed. Or being able to set a priority for what icons get hidden first (and ideally overflowed to the Ice Bar).

**Comment by scollovati** [comment]
Due to the presence of the notch, the icons cannot overflow to the left. This makes them invisible even if the application is active in the background. See here, where I can see 6 out of 8 apps in the bar:

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/b97ad922-1a1a-42ec-994c-498a5be32853" />
